builder/vsphere/driver/vm.go

```diff
@@ -75,7 +75,7 @@ type CreateConfig struct {
 	Datastore     string
 	GuestOS       string // example: otherGuest
 	NICs          []NIC
-	USBController bool
+	USBController []string
```

Comment:
just to remember that this is backward compatible for JSON templates but not HCL2. For now, this is not a problem but people are using HCL2 templates more and more and at some point, we need to consider the backward compatibility for that too.

Reply:
do the fixers even work for HCL2 templates?

Reply:
HCL2 does not have fixers yet, we plan on working on a solution when HCL2 is more stable